client/src/www/model/platform_error.ts

```diff
@@ -205,5 +205,7 @@ export const INTERNAL_ERROR: ErrorCode = 'ERR_INTERNAL_ERROR';
 export const FETCH_CONFIG_FAILED: ErrorCode = 'ERR_FETCH_CONFIG_FAILURE';
```

Comment:
No need to change in this PR, but technically only the error codes here that are exposed to the user should be in the model. The rest is implementation detail that is not exposed to the user, so it doesn't belong in the domain model.

Reply:
Sounds good, this error is not used for now. But I plan to use it when implementing the Go fetching.